packages/mdc-floating-label/_mixins.scss

```diff
@@ -67,3 +67,9 @@
     animation: mdc-floating-label-shake-float-above-#{$modifier} 250ms 1;
   }
 }
+
+@mixin mdc-floating-label-max-width($max-width) {
+  .mdc-floating-label {
```

Comment:
Does it really make sense for this selector to be baked into this mixin? Would this not ordinarily be applied to a selector that already references the floating label root element? (See also the ink/fill color mixins)

Reply:
Updated